cli/Dockerfile-bin

```diff
@@ -9,6 +8,13 @@ COPY controller/gen controller/gen
 COPY controller/util controller/util
 COPY pkg pkg
 RUN mkdir -p /out
+
+# Cache builds without version info
+RUN CGO_ENABLED=0 GOOS=darwin  go build -installsuffix cgo -o /out/conduit-darwin  ./cli
+RUN CGO_ENABLED=0 GOOS=linux   go build -installsuffix cgo -o /out/conduit-linux   ./cli
+RUN CGO_ENABLED=0 GOOS=windows go build -installsuffix cgo -o /out/conduit-windows ./cli
+
```

Comment:
Is it better or worse to have `-ldflags -s -w` here? If the `go` tool builds all the debug info and then strips it out when `-ldflags -s -w` are provided then I assume it would be worse to add `-s -w` here. But, if the `go` tool avoids even generating the debug into when `-s -w` are provided then it might be better.